lib/chusaku/parser.rb

```diff
@@ -21,12 +21,13 @@ module Parser
     #
     # @param {String} path - File path to parse
     # @param {Array<String>} actions - List of valid actions for this route
-    # @return {Array<Hash>} Parsed groups of the file
+    # @return {Hash} Parsed groups of the file
     def self.call(path:, actions:)
       groups = []
       group = {}
+      content = IO.read(path)
```

Comment:
Oh interesting! Was looking into the differences between using `File.open` and `IO.read` a bit. Is the main advantage here that we don't have to worry about closing the file later?

Reply:
I think it's more compact, and if I know I only need a content and nothing else I usually use `IO.read`. If I need more complex flow I use `File.open` with block and logic in it.